### PR TITLE
Secure backup fixes

### PR DESCRIPTION
**Pull request by stefanceriu**
* fixed wrong logout alert title when secure backup is enabled
* disable corrections on the recovery key text field
* allow the recovery key confirmation screen to scroll on small screens

### CODE REVIEW

File: ElementX/Sources/Screens/SecureBackup/SecureBackupRecoveryKeyScreen/View/SecureBackupRecoveryKeyScreen.swift [hunk 1]
```diff
@@ -20,15 +20,31 @@ import SwiftUI
...
+                    .padding()
+                    .onChange(of: focused) { newValue in
+                        guard newValue == true else { return }
+                        reader.scrollTo(textFieldIdentifier)
+                    }
+            }
+        }
+        .safeAreaInset(edge: .bottom) {
+            footer
+                .padding([.horizontal, .bottom])
```
Comment (pixlwave) [inline ElementX/Sources/Screens/SecureBackup/SecureBackupRecoveryKeyScreen/View/SecureBackupRecoveryKeyScreen.swift:39]: FWIW, should probably be explicit and put 16 in here

File: ElementX/Sources/Screens/SecureBackup/SecureBackupRecoveryKeyScreen/View/SecureBackupRecoveryKeyScreen.swift [hunk 2]
```diff
@@ -20,15 +20,31 @@ import SwiftUI
...
-            .padding()
-            .interactiveDismissDisabled()
-            .toolbar { toolbar }
-            .toolbar(.visible, for: .navigationBar)
-            .background(Color.compound.bgCanvasDefault.ignoresSafeArea())
-            .alert(item: $context.alertInfo)
+        ScrollView {
+            ScrollViewReader { reader in
+                mainContent
+                    .padding()
```
Comment (pixlwave) [inline ElementX/Sources/Screens/SecureBackup/SecureBackupRecoveryKeyScreen/View/SecureBackupRecoveryKeyScreen.swift:30]: And here.

**Review by pixlwave — APPROVED** [review]
LGTM. FWIW, maybe using `FullscreenDialog` would be sensible here because it adds affordances for AX1 and larger font sizes.

**Comment by stefanceriu** [comment]
> wondering if using FullscreenDialog would be sensible here

I probably would but that's not what design expects so I'm not going to bother with it too much.

**Pull request merged**